modal/cli/entry_point.py

```diff
@@ -42,6 +45,25 @@ def modal(
     pass
 
 
+def setup():
+    # Sanity check the path
+    try:
+        subprocess.run(["modal", "--help"], capture_output=True)
+        # TODO(erikbern): check returncode?
```

Comment:
Can also do `except PermissionError: ...` to handle case where file is present but not executable.

Reply:
sure good idea